frontend/.vscode/settings.json

```diff
@@ -34,6 +34,9 @@
     "**/bower_components": true,
     "**/.cache-loader": true,
     "**/public/dist": true
+  },
```

Comment:
nice, but should it be in this PR ?

Reply:
this was automatically changed by VS Code..everytime i revert the changes it creates the patch again immediately. does it happen to anyone else?

Reply:
Ref: https://github.com/openshift/console/pull/3803